src/Components/MovieDetails/MovieDetails.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC, useContext } from 'react';
+import { AppContext } from '../../Context/AppContext';
+// @ts-ignore
+import searchButton from '../../Images/Search-Button.svg';
+import { TselectedMovie } from '../../Models/models';
+import { Logo } from '../Logo/Logo';
+import './MovieDetails.scss';
+
+interface Props {
```

Comment:
Be consistent with Props. If you use types almost everywhere, then it's more obvious to use them here too